signac_dashboard/templates/jobs_grid.html

```diff
@@ -17,20 +17,20 @@
 
 
 {% block panels %}
-{% if num_enabled_modules <= 1 %}
+{% if num_enabled_modules < 1 %}
 <section class="section panel">
     <div class="columns is-mobile is-multiline">
 {% endif %}
 {% for job_details in g.jobs %}
 {% set card_count = [] %}
-{% if num_enabled_modules > 1 %}
+{% if num_enabled_modules >= 1 %}
 <section class="section panel" id="{{ job_details.job._id }}">
 {% endif %}
-    {% if ( num_enabled_modules > 1 and g.jobs | length > 1 ) or ( g.jobs | length == 1 and query is defined ) %}
+    {% if ( num_enabled_modules >= 1 and g.jobs | length >= 1 ) or ( g.jobs | length < 1 and query is defined ) %}
```

Comment:
I would evaluate this condition more thoroughly since it seems that the original purpose is not desired any more. The second part looks to see if only one job is matched. I don't know if it make sense if no job is matched.